Tests/TracingTests/InstrumentationSystemTests.swift

```diff
@@ -0,0 +1,40 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Tracing open source project
+//
+// Copyright (c) 2020 Moritz Lang and the Swift Tracing project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+@testable import Instrumentation
+import Tracing
+import XCTest
+
+final class InstrumentationSystemTests: XCTestCase {
+    func testItProvidesAccessToATracer() {
+        let tracer = TestTracer()
+
+        XCTAssertNil(InstrumentationSystem.tracer(of: TestTracer.self))
+
+        InstrumentationSystem.bootstrapInternal(tracer)
```

Comment:
good, was just about to comment on this, previous test version used `bootstrap()` which would have been wrong;

do we also need a way to "unset" it after the test runs or not hmmmm?

Reply:
Yeah, should we just make the `bootstrapInternal` method accept an optional? Or is there an opposite better describing it?

Reply:
accepting optional sounds good to me, good enough for our tests and similar use cases for a "reset" 👍

Reply:
Done ✅